File: SGA_Api/SGA_Api/Models/Traspasos/CompletarTraspasoDto.cs
namespace SGA_Api.Models.Traspasos
{
    public class CompletarTraspasoDto
    {
        public string CodigoAlmacenDestino { get; set; } = "";
        public string UbicacionDestino { get; set; } = "";
        public DateTime FechaFinalizacion { get; set; }
        public int UsuarioFinalizacionId { get; set; }
    }
} 

File: SGA_Android/app/src/main/java/com/example/sga/data/dto/traspasos/CompletarTraspasoDto.kt
package com.example.sga.data.dto.traspasos

data class CompletarTraspasoDto(
    val codigoAlmacenDestino: String,
    val ubicacionDestino: String,
    val fechaFinalizacion: String, // formato ISO 8601: yyyy-MM-ddTHH:mm:ss
    val usuarioFinalizacionId: Int
)
